.storybook/preview-head.html

```diff
@@ -0,0 +1,7 @@
+<meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover" />
+<style>
+  #root {
+    padding: env(safe-area-inset-top) env(safe-area-inset-right)
```

Comment:
Btw, the [safe-area-inset paddings](https://github.com/sumup-oss/circuit-ui/pull/1401/files#diff-2fbc7ad1ca99841f688673a8b6bd5a042f76429ad4d858b14d4c028203f85e94R80) on the side panel container lead to some very weird looking results on landscape mobile. Is this what we're going for @robinmetral?

<img width="599" alt="Screenshot 2022-02-09 at 14 56 39" src="https://user-images.githubusercontent.com/9676303/153205641-25d38eb9-3a2d-4c02-9dd8-2885bfcd9113.png">

Reply:
Hm, interesting 🤔 I'd just trust the env variables and stick with it (maybe that's to handle a screen rotated the other way?), especially since that's just for our Storybook, but we might want to test it with real devices in our apps